relay-event-normalization/src/normalize/processor.rs

```diff
@@ -0,0 +1,1495 @@
+use std::collections::hash_map::DefaultHasher;
```

Comment:
Even though it's a private module, should we add a module doc comment here?

Reply:
I'd defer documenting the undocumented modules of the crate until we've moved all the processors, as the structure may change significantly.

Reply:
I would still add docs here, to at least clarify the general idea behind this normalize processor.

Reply:
Added some vague comments on [4a26e79](https://github.com/getsentry/relay/pull/2668/commits/4a26e79eb0155cebb3a895b14fcb8979f0d2b30f), I'll elaborate more once we've purged light normalization and moved all processing steps.